kovri.md

```diff
@@ -32,13 +32,14 @@ So what can Alice do to mitigate these threats? Well, she can use Kovri!
 If Alice sends her transaction with Kovri, nodes will no longer see her IP address, making passive surveillance tactics impractical.
 
 Now let’s imagine a different scenario.
-Suppose Charlie has an Internet Service Provider that does not want him to use Monero and he receives a warning to cease and desist in the mail.
-Charlie could optionally mask his Internet traffic through Kovri and in doing so, prevent others from knowing he is running a Monero node.
+Suppose Alex wants to support the Monero network by running a full node at her home.
+After a few weeks, she receives a cease and desist letter from her Internet Service Provider informing her that a running node is a violation of the terms of service.
+Alex could have avoided this situation by masking her Internet traffic with Kovri.
 This substantially improves Monero's censorship resistance by making it even more difficult to detect and shut down nodes.
 
-In our last scenario, suppose David is the operator of a mining pool that donates a portion of block rewards to a political party or controversial cause.
+In our last scenario, suppose Andrew is an operator of a mining pool that donates a portion of block rewards to a political party or controversial cause.
 Other nodes could purposefully reject his solved blocks to express their disagreement with his political views.
-However, if David can send his solved blocks through Kovri, then no one can censor the work from his pool, since the source of solved blocks becomes unknown.
+However, if Andrew can send his solved blocks through Kovri, then no one can censor the proof of work from his pool, since the source of solved blocks are unknown.
```

Comment:
Thoughts on changing this to "no one can censor the new blocks created by his pool"?